packages/create-react-app-demo/src/docs-chat/ai.tsx

```diff
@@ -0,0 +1,79 @@
+/** @jsx LLMx.createElement */
+/** @jsxFrag LLMx.Fragment */
+/* eslint-disable react/jsx-key */
+import * as LLMx from 'ai-jsx';
+import React, { useEffect, useRef } from 'react';
+import { DocsQA, ScoredChunk, LocalCorpus, defaultChunker, staticLoader } from 'ai-jsx/batteries/docs';
+import { memo } from 'ai-jsx/core/memoize';
+// @ts-expect-error
+import markdownPath from './brand-new.md';
+import { atom, useAtom } from 'jotai';
+
+import _ from 'lodash';
+
+export class ChatMessage {
+  type: string;
+  content: string;
+  constructor({ type, content }: { type: string; content: string }) {
+    this.type = type;
+    this.content = content;
+  }
+}
+
+export const conversationAtom = atom<ChatMessage[]>([]);
+export const modelCallInProgress = atom<boolean>(false);
+
+// For now, we load the ai.jsx docs from a local markdown file. Once we have a HTML->Markdown converter,
+// we can load the docs from the site directly.
+const docText = await fetch(markdownPath).then((res) => res.text());
```

Comment:
```suggestion
const docResponse = await fetch(markdownPath)
const docText = await docResponse.text()
```

A little cleaner IMO but nbd